app/components/submission/WithCurrentSubmission.js

```diff
@@ -132,13 +140,24 @@ class WithCurrentSubmission extends React.Component {
       return <ErrorPage error={error} />
     }
 
-    return (
-      <Mutation mutation={UPDATE_SUBMISSION}>
-        {updateSubmission =>
-          this.props.children(updateSubmission, initialValues)
-        }
-      </Mutation>
-    )
+    function updateSubmission({ variables }) {
+      return this.props.client.mutate({
+        mutation: UPDATE_SUBMISSION,
+        variables,
+      })
+    }
+
+    function finishSubmission({ variables }) {
+      return this.props.client.mutate({
+        mutation: FINISH_SUBMISSION,
+        variables,
+      })
+    }
```

Comment:
If you defined these as arrow functions you wouldn't need to `bind` them.

Reply:
Is that a preferred approach? Why? I was hesitating between the two.

Reply:
See the top voted comment on that article for an accurate representation of my opinion on the issue. 

But I wasn't talking about class properties. I just meant to convert the local function statements into local arrow function expressions. Just because it's a bit simpler.

Reply:
In this case, since the functions are short (only one instructions) I prefer arrow functions so you have the code in the same place. If the functions were long I would define them somewhere else since this makes the code harder to read (my opinion).